egs/commonvoice/asr1/conf/tuning/train_conformer_standard.yaml

```diff
@@ -0,0 +1,48 @@
+# network architecture
```

Comment:
what does "standard" mean?
If this means Pencheng's standard setup, I think you can just say `train_conformer.yaml`